src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/HistoryOperationsBolt.java

```diff
@@ -58,7 +58,7 @@ public HistoryOperationsBolt(PersistenceManager persistenceManager) {
     public void init() {
         super.init();
 
-        historyService = new HistoryService(persistenceManager);
+        historyWriterService = new HistoryWriterService(persistenceManager);
```

Comment:
hmm, actually it's not the best name. I just wanted to have different names for the service that is invoked by a client and a service that does persistence job on the other side of the messaging. I'll try to come up with 
a better naming.